modules/ossm-release-2-5-8.adoc

```diff
@@ -0,0 +1,31 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-5-8_{context}"]
+= {SMProductName} version 2.5.8
+
+This release of {SMProductName} is included with the {SMProductName} Operator 2.6.5 and is supported on {product-title} 4.14 and later. This release addresses Common Vulnerabilities and Exposures (CVEs).
+
+[id=ossm-release-2-5-8-components_{context}]
+== Component updates
+
+|===
+|Component |Version
+
+|Istio
+|1.18.7
+
+|Envoy Proxy
+|1.26.8
+
+|Kiali Server
+|1.73.18
+|===
+
+[id="ossm-fixed-issues-2-5-8_{context}"]
+== Fixed issues
+
+* https://issues.redhat.com/browse/OSSM-8608[OSSM-8608] Previously, terminating a CNI pod during the installation phase while copying binaries could leave Istio-CNI temporary files on the node's file system. Repeated occurrences could eventually fill up the node's disk space. Now, while terminating a CNI pod during the installation phase firstly deletes existing temporary files before copying the CNI binary, ensuring that only one temporary file per Istio version exists on the node's file system.
```

Comment:
```suggestion
* https://issues.redhat.com/browse/OSSM-8608[OSSM-8608] Previously, terminating a Container Network Interface (CNI) pod during the installation phase while copying binaries could leave Istio-CNI temporary files on the node file system. Repeated occurrences could eventually fill up the node disk space. Now, while terminating a CNI pod during the installation phase, existing temporary files are deleted before copying the CNI binary, ensuring that only one temporary file per Istio version exists on the node file system.
```
per the guideline it's ok to use '_node file system or node disk space'_

https://www.ibm.com/docs/en/ibm-style?topic=grammar-possessives

> Do not use possessive ’s with inanimate objects.